assets/src/dashboard/app/views/shared/useTelemetryOptIn.js

```diff
@@ -20,14 +20,14 @@
 import { useCallback, useState, useEffect, useRef } from 'react';
 import { enableTracking, disableTracking } from '@web-stories-wp/tracking';
 import { useSnackbar } from '@web-stories-wp/design-system';
+import { localStore } from '@web-stories-wp/story-editor';
```

Comment:
Somehow missed that the dashboard was using this. Wonder what we should do about it. Perhaps move that util to design system?

Reply:
Personally don't like that using design system as a utils package. New package instead, or just a utils package?

Reply:
Yeah I hear you. It's certainly not ideal, but also not a huge deal to put it in there until we have a better solution.

I don't like a throw-all `utils` package either, but this also doesn't warrant its own package. Then I'd rather just completely _copy_ this `localStore` util to the dashboard. But comparing that option to just moving to design system, I'd prefer the design system in this case.

Reply:
Okay, so does the design-system contain all the common utils? Should we also move`base64Encode` to design-system then?

https://github.com/google/web-stories-wp/blob/07cc82c3dcff1f13fc739f9958f953ce23ce39e6/packages/story-editor/src/index.js#L21

Reply:
Great question!

To give some background:

Some Web Application Firewalls and WordPress security plugins have had issues with the HTML markup that we send as part of the `content` field in the REST API requests & responses.
That's why we've added this workaround of simply base64-encoding the content during transmission & decode it in the WordPress plugin (see `Decoder` class)

So it's specific to the WordPress plugin and tightly coupled to the REST API interaction.

For this reason, it would be better suited to live in the `wp-story-editor` package.

But since the dashboard currently also uses this util, it would be actually a good candidate for a shared `wp-*` package used by both dashboard and editor.

(Alternatively, refactor dashboard so that this util is no longer needed)

Until we're advanced further with the decoupling, I suppose we can keep it in `story-editor` _for now_.

Reply:
Was trying to move `localStore`, but it's being used at too many places and would deserve a separate PR which I will create after this is merged.

Reply:
Done in https://github.com/google/web-stories-wp/pull/8637